src/components/Student/Student.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div class="cuadro">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12" style="margin-left: 14%;">
+          <div class="card">
+            <div class="card-body">
+              <h5 class="card-title text-uppercase mb-0">Manage Students</h5>
+            </div>
+            <div class="table-responsive">
+              <table class="table no-wrap user-table mb-0"> 
+                  <ul>
+                    <tr>
+                    <th scope="col" class="border-0 text-uppercase font-medium pl-4">#</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Name</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Email</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Course</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Tutor Name</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Tutor Phone</th>
+                    <th scope="col" class="border-0 text-uppercase font-medium">Manage</th>
+                  </tr>
+                    <tr v-for="(student, index) in students" :key="index">
+                      <th> {{index}} </th>
+                      <td> {{student.name}}  </td> 
+                      <td> {{student.email}} </td> 
+                      <td> {{student.course}} </td> 
+                      <td> {{student.tutorName}} </td> 
+                      <td> {{student.tutorPhone}} </td>
+                      <td>
+                      <button
+                        type="button"
+                        class="btn btn-outline-info btn-circle btn-lg btn-circle ml-2"
+                      >
+                        <i class="fa fa-trash"></i>
+                      </button>
+                      <button
+                        type="button"
+                        class="btn btn-outline-info btn-circle btn-lg btn-circle ml-2"
+                      >
+                        <i class="fa fa-edit"></i>
+                      </button>
+                      <button
+                        type="button"
+                        class="btn btn-outline-info btn-circle btn-lg btn-circle ml-2"
+                      >
+                        <i class="fa fa-upload"></i>
+                      </button>
+                    </td>
+                    </tr>
+                  </ul>
+              </table>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+
```

Comment:
Only users that are logged can use this component, so you have to create a method computed that watches if someone is logged